sysml.library/Domain Libraries/Analysis/StateSpaceRepresentation.sysml

```diff
@@ -0,0 +1,76 @@
+package StateSpaceRepresentation {
+    import ISQ::TimeValue;
+
+    // TODO: StateSpace should be a vector (rank-one tensor)
+    abstract attribute def StateSpace;
+    abstract attribute def Input;
+    abstract attribute def Output;
+   
+    abstract calc def NextState(input: Input, stateSpace: StateSpace, timeStep: TimeValue): StateSpace;
+    abstract calc def GetOutput(input: Input, stateSpace: StateSpace): Output;
+    
+    /** Events to be received. */
+    abstract attribute def StateSpaceEvent;
+    attribute def ZeroCrossingEvent :> StateSpaceEvent;
+    
+    /** Item for SSR connection */
+    item def StateSpaceItem;
+    
+    /** StateSpaceDynamics is the simplest form of State Space Representation,
+        and nextState directly computes the stateSpace of the next timestep. */
+    abstract action def StateSpaceDynamics {
+    	in attribute input: Input;
+    	
+        abstract calc nextState: NextState;
+        abstract calc getOutput: GetOutput;
+        attribute stateSpace: StateSpace;
+
+        // To be revised when bind allows an expression
```

Comment:
Thank you for your improvement.  Maybe Line28 in StateSpaceRepresentation.sysml is no longer needed by your changes.

Reply:
I removed the comment.